src/components/activity-history-list/contact-details-record/contact-details-record.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { ComponentPropsWithoutRef } from 'react';
+import { locale } from '@services';
+import { Activity, ActivityChangeRecord } from '../../../services/activities';
+import {
+    ActivityRecordItem,
+    MigratedEntityRecord,
+    UpdatedEntityRecord,
+    formattedDate,
+    updatedRecord,
+} from '../';
+
+const { activities }: any = locale;
```

Comment:
`: any`, what is that for?

Reply:
Removed `: any` as it is of no use

Reply:
TS complains otherwise for dynamically trying to get locale for an entity as
`const activitiesOnTargetType = activities[targetType];`